.github/workflows/python-package.yaml

```diff
@@ -28,26 +28,30 @@ jobs:
       -
         name: Install dependencies and the package
         run: |
-          apt-get update -q && apt-get install -qy python3-pip python-is-python3 ros-$ROS_DISTRO-example-interfaces
+          apt-get update -q && apt-get install -qy python3-pip python-is-python3 python3-colcon-ros ros-$ROS_DISTRO-example-interfaces
           pip install --upgrade pip setuptools
-          python -m pip install -e '.[dev]'
+          python -m pip install -e './ros2_easy_test[dev]'
```

Comment:
Typically a ros package is installed or used by adding it in a workspace under the `src` directory.
Once the directory structure is established and dependencies are moved to package.xml, we should be able to run something like:
```
#Install dependencies
rosdep update
rosdep install --from-paths src -y --ignore-src --rosdistro humble

# Build
colcon build

# Source the package
source install/setup.bash

# Test
colcon test
```
Alternatively you could use a predefined action like: [ros-tooling/action-ros-ci@master](https://github.com/ros-tooling/action-ros-ci)